test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10811,7 +10811,9 @@ def apply_config(self, config=None, validate=True, path=None):
         if validate:
             self.signal('-HUP')
             try:
-                self.log_match("Error reading line", n=10,
+                self.log_match("Sched;reconfigure;Scheduler is reconfiguring",
+                               n=10, starttime=reconfig_time)
+                self.log_match("Error reading line", n=10, max_attempts=10,
```

Comment:
Once you see the first log message, 'Error reading line' messages should be logged pretty much immediately.  I'd set max_attempts to 2 at most.  It used to be 1.  I'd suggest leaving it there.

Reply:
@bhroam do you know why do we need to check for that message at first place?

Reply:
Set it to 2 so that it gets at least half a second. I assume the first check is immediate and the second happens after half a second has elapsed.

Reply:
@anamikau, that is the message that shows up in the scheduler logs if there is an invalid sched_config parameter.  If you want to tell if there are any errors in the sched_config, then that's the log message to search for.  Since we're reverting to a known valid state, is it worthwhile even checking for that?  It will add .5s to each test run.

Reply:
I had a discussion with a few people in the office.  We all agreed that since we're reverting to a known valid configuration, there is no reason to check for errors.  I'd suggest leaving the change you made, but also make a change to the scheduler's revert_to_defaults() to call apply_config() with validate=False.  This will avoid checking for errors in setUp().

Reply:
The Server, Scheduler, and MoM classes are all defined in pbs_testlib.py. The scheduler's version of revert_to_defaults never calls Scheduler.apply_config, nor does it call Scheduler.set_sched_config (which does call Scheduler.apply_config). I'm not sure what you're asking for.

Reply:
I was reading the wrong revert_to_defaults().  The mom revert calls her apply_config().  It's definitely the scheduler's apply_config() that is causing the setUp() delay.  I don't know where it is being called.  Wherever it is called, I still think we can pass validate=False.